README.md

```diff
@@ -249,7 +257,200 @@ AdyenCardComponent(
+  sessionId: sessionResponse.id,
+  sessionData: sessionResponse.sessionData,
+  configuration: googlePayComponentConfiguration,
+);
+```
+4. Get the Google Pay payment method from the `sessionCheckout` object.
+5. Use the Google Pay component widget:
+```dart
+AdyenGooglePayComponent(
+  configuration: googlePayComponentConfiguration,
+  paymentMethod: paymentMethod,
+  checkout: sessionCheckout,
+  loadingIndicator: const CircularProgressIndicator(),
+  onPaymentResult: (paymentResult) {
+    //Handle the payment result
+  },
+),
+``` 
+</p>
+
+### Google Pay Component with Advanced flow
+
+1. From your backend, make a [`/paymentMethods`](https://docs.adyen.com/api-explorer/Checkout/71/post/paymentMethods) request.
+2. Get the Google Pay payment method by filtering the payment methods list.
+3. Create the `GooglePayComponentConfiguration`. It requires the `GooglePayConfiguration` which contains the Google Pay environment. You can also provide optional properties for an enhanced Google Pay use case.
+
+```dart
+ final GooglePayComponentConfiguration googlePayComponentConfiguration = GooglePayComponentConfiguration(
+  // Change the environment to live when you are ready to accept real payments.
+  environment: Environment.test,
+  clientKey: CLIENT_KEY,
+  countryCode: COUNTRY_CODE,
+  amount: AMOUNT,
+  googlePayConfiguration: const GooglePayConfiguration(
+    // Change the environment to live when you are ready to accept real payments.
+    googlePayEnvironment: GooglePayEnvironment.test,
+    shippingAddressRequired: true,
+  ),
+);
+```
+4. Create an `AdvancedCheckoutPreview` providing two callbacks:
+- `onSubmit`: from your backend, make a [`/payments`](https://docs.adyen.com/api-explorer/Checkout/latest/post/payments) request. The callback returns two parameters:
+  - `data`: payment data that needs to be forwarded.
+  - `extra`: extra information (e.g. shipping address) in case it is specified in the Google Pay configuration. Can be null if not needed.
+- `onAdditionalDetails`: from your backend, make a [`/payments/details`](https://docs.adyen.com/api-explorer/Checkout/71/post/payments/details)
+```dart
+final AdvancedCheckoutPreview advancedCheckout = AdvancedCheckoutPreview(
+  onSubmit: YOUR_ON_SUBMIT_CALL,
+  onAdditionalDetails: YOUR_ON_ADDITIONAL_DETAILS_CALL,
+);
+```
+
+5. Create the Google Pay component widget:
+```dart
+ AdyenGooglePayComponent(
+  configuration: googlePayComponentConfiguration,
+  paymentMethod: paymentMethod,
+  checkout: advancedCheckout,
+  style: GooglePayButtonStyle(width: 250, cornerRadius: 4),
+  loadingIndicator: const CircularProgressIndicator(),
+  onPaymentResult: (paymentResult) {
+    //Handle the payment result
+  },
+),
+``` 
+
+</p>
+
+### Multi Component setup
 
+The SDK currently supports component combination in the advanced flow only. By using this flow, you can use the card component together with the Google Pay or Apple Pay component.
```

Comment:
Not sure if I understand this correctly. "Use" is a vague word here, can they integrate with all but will only be able to use one component for each payment session? Or would it not be possible to create a configuration that includes all components for Sessions? 

Maybe something like: 
For a multi component setup, for example to display the card component together with the Google Pay or Apple Pay component in your checkout, you need to use the Advanced flow. 

If it is not about what's included in the checkout page session based, but about the integration all together, say "integrate" instead of "display".